### PR TITLE
Added Top Food Card

### LINKED ISSUE #169: Update "Top food items" card
- Replace sample data with https://nercacuk.sharepoint.com/sites/BGSBMGFMAPS/Shared%20Documents/WP4%20-%20Tool%20Development/top-foods.json

- Default tab should be a treemap representation of this data

- Second tab displaying tabular version

### PR DESCRIPTION
**Pull request by Fuhji**
- Includes table view of data 
- Includes treeview of data
- updated top-food.json data to use preferred data structure

closes #169

### CODE REVIEW

File: src/app/pages/quickMaps/quickMaps.service.ts [hunk 1]
```diff
@@ -65,17 +86,25 @@ export class QuickMapsService {
...
-    if (force || (this.countryId !== countryId)) {
+    if (force || this.countryId !== countryId) {
       this.countryIdSrc.next(countryId);
     }
   }
 
   public get micronutrientId(): string {
     return this.micronutrientIdSrc.value;
   }
+  public get micronutrientDict(): DictionaryItem[] {
```
Comment (jon571) [inline src/app/pages/quickMaps/quickMaps.service.ts:101]: This new function doesn't look like it's used.

**Comment by jon571** [comment]
Never heard of pigeon peas!

File: src/app/pages/quickMaps/quickMaps.service.ts (hunk 1 above)
Comment (Fuhji) [reply src/app/pages/quickMaps/quickMaps.service.ts:101]: > Never heard of pigeon peas!

or cowpeas!

**Comment by Fuhji** [comment]
I really like the styling on the chart of table - purty!

File: src/app/pages/quickMaps/quickMaps.service.ts (hunk 1 above)
Comment (Fuhji) [reply src/app/pages/quickMaps/quickMaps.service.ts:101]: Its used https://github.com/micronutrientsupport/micronutrient-support-tool/pull/206/files/0eeb0ac28af2e4ed9f637a793e0488a963c5e75e#diff-e263941c853363f7d21160ec7fe64e7b10ae31de2e41501f758dc2803a109f57R35

File: src/app/pages/quickMaps/pages/baselineDetails/foodItems/foodItems.component.ts [hunk 2]
```diff
@@ -1,53 +1,92 @@
...
 
-  constructor() { }
+  constructor(private currentDataService: CurrentDataService, private quickMapsService: QuickMapsService) {}
 
   ngOnInit(): void {
-    this.dataSource = new MatTableDataSource(this.desserts);
+    void this.currentDataService
+      .getTopFood(
+        this.quickMapsService.countryDict,
+        this.quickMapsService.micronutrientDict,
```
Comment (Fuhji) [inline src/app/pages/quickMaps/pages/baselineDetails/foodItems/foodItems.component.ts:35]: Here @jon571

Comment (jon571) [reply src/app/pages/quickMaps/pages/baselineDetails/foodItems/foodItems.component.ts:35]: github didn't find it as a reference :-(

**Comment by jon571** [comment]
Looks good to me!

**Pull request merged**